pom.xml

```diff
@@ -313,4 +227,101 @@
       </plugins>
     </pluginManagement>
   </build>
+
+  <profiles>
+    <profile>
```

Comment:
I think we need to add the deploy profile at the same time. 
You can find the travis deploy command [here](https://github.com/apache/incubator-servicecomb-java-chassis/blob/master/scripts/travis.sh#L59)

Reply:
there is no any log for maven-deploy-plugin